src/nvim/file_search.c

```diff
@@ -749,7 +749,7 @@ char_u *vim_findfile(void *search_ctx_arg)
          * Expand wildcards like "*" and "$VAR".
          * If the path is a URL don't try this.
          */
-        if (path_with_url(dirptrs[0])) {
+        if (path_with_url((char *)dirptrs[0])) {
           stackp->ffs_filearray = (char_u **)xmalloc(sizeof(char *));
```

Comment:
> I'm willing to be overridden on this, but I think that's outside the scope of this PR.

Yes, it might be outside this PRs scope, but it's an arguably innocuous change which will probably be forgotten if not applied here (due to how small it is). I think a "passing by" commit would be useful.